.travis.yml

```diff
@@ -8,7 +8,7 @@ matrix:
     - env: js-tests
       language: node_js
       # The Node version here must be kept in sync with that in `package.json`.
-      node_js: '12'
+      node_js: '12.12.0'
```

Comment:
This change is only because I saw an issue with Travis failing to pick up the right version of Node (not available remotely).
This is in *hopes* to pick up a cached version.